pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -248,9 +248,9 @@ ARG PT_TORCHDATA_URL
 
 # Install SM AWS-PyTorch (force remove will increase the docker image size drastically)
 RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
-  torchvision torchaudio torchtext cpuonly \
+  torchvision torchaudio torchtext cpuonly aws-ofi-nccl-dlc \
```

Comment:
Why is aws-ofi-nccl needed on a CPU image that doesn't actually have NCCL installed?